src/mgr/DaemonServer.cc

```diff
@@ -1050,6 +1050,69 @@ bool DaemonServer::handle_command(MCommand *m)
       });
     cmdctx->reply(r, "");
     return true;
+  } else if (prefix == "osd pool stats") {
+    string pool_name;
+    cmd_getval(g_ceph_context, cmdctx->cmdmap, "pool_name", pool_name);
+    
+    int64_t poolid = -ENOENT;
+    bool one_pool = false;
+    r = cluster_state.with_pgmap([&](const PGMap& pg_map) {
+        return cluster_state.with_osdmap([&](const OSDMap& osdmap) {
+	  if (!pool_name.empty()) {
+	      poolid = osdmap.lookup_pg_pool_name(pool_name);
+
+	    if (poolid < 0) {
+	      assert(poolid == -ENOENT);
+	      ss << "unrecognized pool '" << pool_name << "'";
+	      return -ENOENT;
+	    }
+	    one_pool = true;
+	  }
+	  
+	  stringstream rs;
+
+	  if (f)
+	    f->open_array_section("pool_stats");
+	  else
+	  {
```

Comment:
move the `{` up to the end of previous line.

Reply:
done, with following patch:

```diff
diff --git a/src/mgr/DaemonServer.cc b/src/mgr/DaemonServer.cc
index d6a201bfff..56dedb8c99 100644
--- a/src/mgr/DaemonServer.cc
+++ b/src/mgr/DaemonServer.cc
@@ -1073,17 +1073,14 @@ bool DaemonServer::handle_command(MCommand *m)
 
 	  if (f)
 	    f->open_array_section("pool_stats");
-	  else
-	  {
-	    if (osdmap.get_pools().empty())
-	    {
+	  else {
+	    if (osdmap.get_pools().empty()) {
 	      ss << "there are no pools!";
 	      goto stats_out;
 	    }
 	  }
 
-	  for (auto &p : osdmap.get_pools())
-	  {
+	  for (auto &p : osdmap.get_pools()) {
 	    if (!one_pool) {
 	      poolid = p.first;
             }
@@ -1096,13 +1093,11 @@ bool DaemonServer::handle_command(MCommand *m)
 	  }
 
 	stats_out:
-	  if (f)
-	  {
+	  if (f) {
 	    f->close_section();
 	    f->flush(cmdctx->odata);
 	  }
-	  else
-	  {
+	  else {
 	    cmdctx->odata.append(rs.str());
 	  }
 	  return 0;
diff --git a/src/mon/PGMap.cc b/src/mon/PGMap.cc
index 24816094d8..8501fa3ce6 100644
--- a/src/mon/PGMap.cc
+++ b/src/mon/PGMap.cc
@@ -2123,8 +2123,7 @@ void PGMap::dump_filtered_pg_stats(ostream& ss, set<pg_t>& pgs) const
 void PGMap::dump_pool_stats_and_io_rate(int64_t poolid, const OSDMap &osd_map, Formatter *f, stringstream *rs) const {
   string pool_name = osd_map.get_pool_name(poolid);
 
-  if (f)
-  {
+  if (f) {
     f->open_object_section("pool");
     f->dump_string("pool_name", pool_name.c_str());
     f->dump_int("pool_id", poolid);
@@ -2134,14 +2133,12 @@ void PGMap::dump_pool_stats_and_io_rate(int64_t poolid, const OSDMap &osd_map, F
   list<string> sl;
   stringstream tss;
   pool_recovery_summary(f, &sl, poolid);
-  if (!f && !sl.empty())
-  {
+  if (!f && !sl.empty()) {
     for (auto &p : sl)
       tss << "  " << p << "\n";
   }
 
-  if (f)
-  {
+  if (f) {
     f->close_section(); // object section recovery
     f->open_object_section("recovery_rate");
   }
@@ -2151,8 +2148,7 @@ void PGMap::dump_pool_stats_and_io_rate(int64_t poolid, const OSDMap &osd_map, F
   if (!f && !rss.str().empty())
     tss << "  recovery io " << rss.str() << "\n";
 
-  if (f)
-  {
+  if (f) {
     f->close_section(); // object section recovery_rate
     f->open_object_section("client_io_rate");
   }
@@ -2165,10 +2161,8 @@ void PGMap::dump_pool_stats_and_io_rate(int64_t poolid, const OSDMap &osd_map, F
 
   // dump cache tier IO rate for cache pool
   const pg_pool_t *pool = osd_map.get_pg_pool(poolid);
-  if (pool->is_tier())
-  {
-    if (f)
-    {
+  if (pool->is_tier()) {
+    if (f) {
       f->close_section(); // object section client_io_rate
       f->open_object_section("cache_io_rate");
     }
@@ -2179,8 +2173,7 @@ void PGMap::dump_pool_stats_and_io_rate(int64_t poolid, const OSDMap &osd_map, F
     if (!f && !rss.str().empty())
       tss << "  cache tier io " << rss.str() << "\n";
   }
-  if (f) 
-  {
+  if (f) {
     f->close_section(); // object section cache_io_rate
     f->close_section(); // object section pool
   } else {
```